Gemfile.lock

```diff
@@ -282,8 +282,5 @@ DEPENDENCIES
   webmock
   will_paginate-bootstrap
 
-RUBY VERSION
-   ruby 2.3.0p0
-
```

Comment:
I believe it's related to this: https://github.com/bundler/bundler/issues/4621

Basically new version of bundler adds in a patch level into the lockfile.